tests/FilterAttachedResults/FilterAttachedResults.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {
+    FilterAttachedResults,
+    IFilterAttachedResultsOptions
+} from '../../src/components/FilterAttachedResults/FilterAttachedResults';
+import { Mock, Simulate } from 'coveo-search-ui-tests';
+
+describe('FilterAttachedResults', () => {
+    let filter: Mock.IBasicComponentSetup<FilterAttachedResults>;
+
+    beforeEach((done) => {
+        filter = Mock.basicResultComponentSetup<FilterAttachedResults>(
+            FilterAttachedResults
+        );
+        setTimeout(done, 0);
```

Comment:
Can you use clock instead of direcxt setTimeout ?
See https://jasmine.github.io/api/2.6/Clock.html.

Reply:
I used the trick of setTimeout with 0 ms to send it to the end of the stack, do you think using clock().tick(1) would be cleaner?

Reply:
Yeah, this way your test is dettach from the main clock.